features/forced-colors.yml

```diff
@@ -0,0 +1,14 @@
+name: Forced colors
+description: The `forced-colors` CSS @media rule and `forced-color-adjust` CSS property control styles when a user has chosen a limited palette of colors in forced colors mode.
+spec:
+  - https://drafts.csswg.org/mediaqueries-5/#forced-colors
+  - https://drafts.csswg.org/css-color-adjust-1/#forced-color-adjust-prop
+group: css
+status:
+  compute_from: css.at-rules.media.forced-colors
```

Comment:
Can you use the feature without `forced-color-adjust`? It seems that merely detecting forced colors mode without being able to change it wouldn't be useful.
I wonder if we should just remove the `compute_from` rule here.

Reply:
You can use it without `forced-color-adjust`. For example, you can add a border to a button if it otherwise gets styles from box-shadow, which is removed in forced colors mode.